### DISCUSSION TITLE
How to get a list of bluetooth devices in termux?

### DISCUSSION THREAD

**Discussion by Vponed** [termux/termux-app#4161] [Q&A]
Unfortunately, I did not find an answer to the question on my own.

**Comment by jcharth** [comment]
may be use python. there are a bunch of scripts in python for bluetooth. i bought a tgam sensor. i will test it with linux first. there is a brainwave script in python to interface bluetooth. crazy code if you ask me.

**Reply by Vponed** [reply]
yeah, but how to use it? 
is there any example code?
